### ISSUE TITLE
Sending notification to topic without registration_ids should send notification to phone

### ISSUE THREAD

**Issue by ericlathrop** [codedge-llc/pigeon#146]
### Environment

* Elixir & Erlang/OTP versions (`elixir --version`): 
```
Erlang/OTP 20 [erts-9.3] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:10] [hipe] [kernel-poll:false]

Elixir 1.8.1 (compiled with Erlang/OTP 20)
```

* Operating system: 
`Ubuntu 18.04`

### Current behavior

This code:
```elixir
    Pigeon.FCM.Notification.new([], %{
      "title" => "your title",
      "body" => "your message"
    })
    |> put_condition("'your-topic' in topics")
    |> Pigeon.FCM.push()
```
does not send any notifications to my app, even though my app subscribes to the `your-topic` topic.

### Expected behavior

I expect the above code to send notifications to my app, where my app subscribes to the `your-topic` topic. I can send to the topic without specifying `registration_ids` in the Firebase console, so it should work here.

**Comment by FranciZ** [comment]
Have you found a solution for this by any chance @ericlathrop ?

**Comment by ericlathrop** [comment]
@FranciZ No, because of this issue I wasn't able to use this project, and instead implemented my own code to send notifications. Unfortunately, I haven't gotten around to open sourcing my library yet.

**Comment by FranciZ** [comment]
@ericlathrop Thanks for the quick response, appreciate it!

**Comment by siddhant3030** [comment]
@ericlathrop Hey. Can you tell me what is registration_id in the firebase console?
